src/session.ts

```diff
@@ -68,6 +68,7 @@ export type Event = EventMetadata & PromptEvent;
 
 export async function send_event(event: Event) {
   if (!event.apiKey) {
+    console.warn(`No LIBRETTO_API_KEY provided, no event will be sent`);
```

Comment:
This helped me again. I think this is worth doing. I think if the person gets to this `save_event` call at all, then it means they have `libretto` in their prompt, and everything else was good to go, so it seems safe to warn them that we are skipping the event.